lib/tictactoe/ai/minimax_ai.rb

```diff
@@ -10,15 +10,27 @@ class MinimaxAi
       DEFAULT_ALPHA = -100
       DEFAULT_BETA = 100
 
-      def calculate_next_move(board, marker, alpha = DEFAULT_ALPHA, beta = DEFAULT_BETA, depth = 6)
+      def calculate_next_move(board, marker)
+        # check 1 move deep for a winning or blocking move
+        # prefer this to a depth first search to improve speed
+        winning_move = next_winning_move(board, marker)
+        return winning_move if winning_move
+        blocking_move = block_next_winning_move(board, marker)
+        return blocking_move if blocking_move
+        best_move = calculate_next_move_score(board, marker)
+        best_move[:move]
+      end
+
+      def calculate_next_move_score(board, marker, alpha = DEFAULT_ALPHA, beta = DEFAULT_BETA, depth = 7)
         minimax_value = MINIMAX_STARTING_VALUE
-        best_move = :NO_BEST_MOVE
+        best_move = {:move => :NO_BEST_MOVE, :score => -1}
```

Comment:
improve readability here. No need for minimax_value variable, just instantiate best_move with MINIMAX_STARTING_VALUE.